index.html

```diff
@@ -376,26 +687,28 @@ <h2 class="top3-1-p">LET’S CONNECT</h2>
     </section>
     <!-- the contact form's section -->
     <section class="form">
-        <div class="contact-me">
-            <h2 class="form-h2"><span>Contact me</span></h2>
-            <p class="txt-p">
-                <span>
```

Comment:
- [ ] Good job adding a proper header on your web page, :clap: , I’ve noticed that there is some overflow with the about me section  , where you are working on screen that have `768px` , Kindly make sure you fix it :+1: , it will make your web page more responsive :+1: 



[Screencast from 02-25-2023 12_05_51 AM.webm](https://user-images.githubusercontent.com/80458199/221304055-53f0b207-9d0f-4ce0-b8f7-ccf778bba305.webm)